test/test_browser.py

```diff
@@ -4758,15 +4758,26 @@ def test_pthread_reltime(self):
 
   # Tests that it is possible to load the main .js file of the application manually via a Blob URL,
   # and still use pthreads.
-  def test_load_js_from_blob_with_pthreads(self):
+  @parameterized({
+    'blob': ('main_js_as_blob_loader.html',False),
+    'url': ('main_js_as_url_loader.html',False),
+    'blob_es6': ('main_js_as_blob_loader_es6.html',True),
+    'url_es6': ('main_js_as_url_loader_es6.html',True),
+  })
+  def test_load_js_from_blob_with_pthreads(self, which_html, es6):
```

Comment:
I guess that `blob` part of this test name is no longer correct?

Should we rename this test to `test_mainScriptUrlOrBlob` like the `test_other.py` test?

Reply:
Since `mainScriptUrlOrBlob` is really only a pthread feature I think you can just call it `test_mainScriptUrlOrBlob`, like in `test_other.py`.     (Or rename the one in test_other.py.. either way they probably should be called the same thing).